packages/iov-multichain/src/signingservercore.ts

```diff
@@ -92,6 +95,7 @@ export class SigningServerCore {
     try {
       authorizedIdentities = await this.authorizeGetIdentities(reason, matchingIdentities, meta);
     } catch (error) {
+      this.logError(error);
```

Comment:
… we don't know what we are catching here. This could be anything and is not in IOV-Core's control. A caller you throw a string or a number or any kind of object that we catch here.